DUnitX.TestFramework.pas

```diff
@@ -249,12 +259,20 @@   TTestFixtureInfoList = class(TDUnitXList<ITestFixtureInfo>, ITestFixtureInfoLi
     function GetExceptionLocationInfo : string;
     function GetExceptionAddressInfo : string;
     function GetExceptionAddress : Pointer;
+    function GetIsComparable: boolean;
+    function GetExpected: string;
+    function GetActual: string;
+    function GetFormat: TDUnitXComparableFormatClass;
 
     property ExceptionClass : ExceptClass read GetExceptionClass;
     property ExceptionMessage : string read GetExceptionMessage;
     property ExceptionLocationInfo : string read GetExceptionLocationInfo;
     property ExceptionAddressInfo : string read GetExceptionAddressInfo;
     property ExceptionAddress : Pointer read GetExceptionAddress;
+    property IsComparable: boolean read GetIsComparable;
+    property Expected: string read GetExpected;
+    property Actual: string read GetActual;
+    property Format: TDUnitXComparableFormatClass read GetFormat;
```

Comment:
I tried that originally, but the lifecycle of the actual instance of the exception is gone by the time it gets to the end of the test (at least I think that is what was happening - I didn't try too many things since I assumed it wasn't going to work).  I also thought that, if I could do that, someone already would've been passing the exception instance instead of breaking out all the info (class, message, etc) out separately.  

What I could do is have an ExtraInfo interface that could be queried - that would also be beneficial when and if I implement binary comparison and/or other "details" about the assertion failure that may be datatype specific.  I'll explore that a little later today.